tests/integration/check-custom-repo/test_custom-repos.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
+
+from envparse import env
+
+
+@pytest.mark.good_tests
+def test_good_convertion(shell):
+    convertion = shell(
+        command=[
+            (
+                "convert2rhel -y "
```

Comment:
You probably should use another command:
```
convert2rhel --disable-submgr --disablerepo "*" --enablerepo rhel-{ version here}-server-rpms --debug
```

Reply:
Yes, in your implementation you have:
```
  if not tool_opts.disable_submgr:
        logger.info("Skipping the check of repositories due to the use of RHSM for the conversion.")
        return
 ```
So you're skipping the check if rhsm not disabled